modules/template/views/admin/editSource.php

```diff
@@ -75,10 +77,10 @@
                     <span class="caret"></span>
                 </button>
                 <ul class="dropdown-menu" id="addElementSelect">
-                    <?php foreach (TemplateElement::getAvailableTypes() as $type) : ?>
+                    <?php foreach ($elementTypeService->getTypeInstances() as $elementType) : ?>
                         <li>
                             <a data-action-click="ui.modal.load" data-action-data-type="json" data-action-url="<?= Url::to(['/custom_pages/template/admin/add-element', 'templateId' => $model->id, 'type' => $type]) ?>" href="#">
```

Comment:
@luke- `$type` is undefined, change to `get_class($elementType)` or set key as class name on init the array in the method `getTypeInstances()`.